app/assets/javascripts/discourse/tests/acceptance/category-new-test.js

```diff
@@ -29,6 +39,13 @@ acceptance("Category New", function (needs) {
       "it transitions to the category edit route"
     );
 
+    await click(".edit-category-nav .edit-category-tags a");
+
+    assert.ok(
+      exists("[data-value='TagGroup1']"),
```

Comment:
Is this attached to any CSS class? Feels a little odd to be only using the data attribute here.

Reply:
Yes. Updated to be more specific.